### DISCUSSION TITLE
Download UX

### DISCUSSION THREAD

**Discussion by WhiteMemory99** [khoadng/Boorusama#316] [Ideas]
I really like your app, but the image download button needs more visual or/and tactile feedback.
When I press the button, I always have some instinctive urge to double-check if the download has actually started, because there is almost nothing indicating that the button was pressed.

Like some toast with "Download started!" (+vibration?) would feel much better.

**Comment by khoadng** [comment]
@WhiteMemory99 
The app will display a notification in the status bar to indicate the download status, but I suppose it might not be clear enough for users to notice. I'll incorporate a toast, as you've suggested, to provide better feedback
